regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -103,6 +103,7 @@ var CommentReviewView = Backbone.View.extend({
       var $elm = $(elm);
       var $text = $elm.find('input');
       var $select = $elm.find('select');
+      var $selectDropdown = $elm.find('.select-dropdown');
```

Comment:
Ah, sorry, trigger happy. I'd recommend not letting the JS be a blocker here; @xtine, @jmcarp, I or someone else can swing back so that you can focus on more pressing stuff.